start-planning-conversation/SKILL.md

```
@@ -1,18 +1,18 @@
 disable-model-invocation: true
 ---
 
 # Start planning conversation
 
 **Usage:** `/start-planning-conversation [url, id, or description]`
 
-Set `/grill-me` up to do its job on a real problem, then hand what it produces to
+Turn a real problem into a shared understanding, then hand that understanding to
 the planning pipeline:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)